spec/lib/baw-workers/harvest/single_file_spec.rb

```diff
@@ -183,10 +183,10 @@
       password = 'password'
       auth_token = 'auth token this is'
 
-      file_hash = 'SHA256::c6d561c91664a92a1598bda3b79734d5ee29266ad0411ffaea1188f29d5b6439'
+      file_hash = 'SHA256::b06458c96ac2de75d77b83e1b3acb3e64b84a4bfe671c794153855bf7b5e96fd'
```

Comment:
Definitely __*do not*__ change the meaning of existing fields. Do this properly. Otherwise it's a quick way to shoot yourself in the foot.

First of all, why do you care what format the file used to be?

If you must store info about pre-conversion, what info do you want to store?
Add new fields to `audio_recording` to store this information. Or, maybe, store it in the existing machine readable notes field.